### DISCUSSION TITLE
Example term mapping exercise: volume

### DISCUSSION THREAD

**Discussion by turbomam** [microbiomedata/nmdc-schema#2022] [Schema Best Practices]
Many qualities of things are modeled very nicely by the Phenotype and Traits Ontology, PATO.

Their textual definitions can be pretty academic, but they are reused by a lot of other systems*, so they carry some weight.

*34 other OBO foundry ontologies as of 2023-10-27 in this case!

So [PATO:0000918](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0000918) would be a good [mapping](https://linkml.io/linkml-model/latest/docs/mappings/) of some kind

**Comment by turbomam** [comment]
Wikidata might be a good low-stress mapping target , too.

See the bottom of the tools pull-down on the left of this screenshot. 

![image](https://github.com/microbiomedata/nmdc-schema/assets/25185273/d45517da-c343-4cb7-aae3-744483e331ba)

**Comment by turbomam** [comment]
https://www.wikidata.org/wiki/Q39297

That even has a [QUDT quantity kind ID](https://www.wikidata.org/wiki/Property:P8393)

**Comment by kheal** [comment]
This is not actionable, converting to a discussion.
